source/blood/src/globals.cpp

```diff
@@ -59,6 +59,10 @@ void _ThrowError(const char *pzFormat, ...)
     vsprintf(buffer, pzFormat, args);
     LOG_F(ERROR, "%s(%i): %s", _module, _line, buffer);
 
+    // break into debugger, so the error can be investigated
+    // (unless NDEBUG is defined; this is handled in build/include/compat.h)
```

Comment:
These comments are unnecessary, they are redundant to the meaning of debug_break. It's like `int i = 2; // set i to 2`.